can_tools/scrapers/official/AZ/AZ_TableauPull.py

```diff
@@ -0,0 +1,280 @@
+import requests
+import urllib.parse
+from bs4 import BeautifulSoup
+from can_tools.scrapers.base import CMU
+from can_tools.scrapers.official.base import StateDashboard
+import pandas as pd
+import json
+import re
+
+
+class ArizonaData(StateDashboard):
+    """
+    Fetch county level covid data from Arizona's Tableau dashboard
+    """
+
+    # Initialize
+    source = "https://www.azdhs.gov/preparedness/epidemiology-disease-control/infectious-disease-epidemiology/covid-19/dashboards/index.php"
+    has_location = True
+    location_type = "county"
+    state_fips = 4
```

Comment:
I know AZ is fips code 4, but could we set the `state_fips` using `int(us.states.lookup("Arizona").fips)`?

We instituted this because we have previously introduced incorrect data by setting them incorrectly (we forgot to change copy-pasted codes etc...)

Reply:
Can do.  I had generated my own FIPS code list for testing purposes since Tableau will need an all-caps list of county names anyways for the subview generation.  But that list can be generated iteratively